config.sh

```diff
@@ -1,7 +1,9 @@
+
+EXCLUDE_IP=${EXCLUDE_IP:=$(hostname -i)}
```

Comment:
in case the user wants to overwrite the exclude with another one. can be removed if you don't want users modifying the value